app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,45 @@
+<resources>
+    <string name="app_name">Medical Helper</string>
+    <string name="email">email</string>
+    <string name="UserId">User Id</string>
+    <string name="Password">Password</string>
+    <string name="ConfirmPassword">Confirm Password</string>
+    <string name="error_message_email">Enter Valid Email</string>
+    <string name="error_message_password">Enter Password</string>
+    <string name="error_password_match">Password doesnot match</string>
+    <string name="facebook_app_id">2030765153863126</string>
+    <string name="fb_login_protocol_scheme">fb2030765153863126</string>
+    <string name="action_emergency">Emergency</string>
+    <string name="action_settings">Settings</string>
+    <string name="action_aboutus">About Us</string>
+    <string name="action_logout">Logout</string>
+
```

Comment:
Please do not leave blank lines. Blank lines are used to logically separate piece of codes, 
example : 
Wrong
int a;
int b; 
int c;
a = b+c;
c = a-b;
char f;


char g;


Right : 
int a;
int b;
int c;

a = b+c;
c = a-b;

char f;
char g;